### ISSUE TITLE
Audio Classification

### ISSUE THREAD

**Issue by abhisheks008** [abhisheks008/ML-Crate#463]
### ML-Crate Repository (Proposing new issue)
:red_circle: **Project Title** : Audio Classification
:red_circle: **Aim** : The aim of this project is to classify audio files.
:red_circle: **Dataset** : https://www.kaggle.com/datasets/khadijehvalipour/audio-classification
:red_circle: **Approach** : Try to use 3-4 algorithms to implement the models and compare all the algorithms to find out the best fitted algorithm for the model by checking the accuracy scores. Also do not forget to do a exploratory data analysis before creating any model.

**********************************************************************************
### 📍 **Follow the Guidelines to Contribute in the Project :**
* You need to create a separate folder named as the Project Title.
* Inside that folder, there will be four main components.
   * Images - To store the required images.
   * Dataset - To store the dataset or, information/source about the dataset.
   * Model - To store the machine learning model you've created using the dataset.
   * `requirements.txt` - This file will contain the required packages/libraries to run the project in other machines.
* Inside the `Model` folder, the `README.md` file must be filled up properly, with proper visualizations and conclusions.

***********************************************************************
:red_circle::yellow_circle: **Points to Note :**

- The issues will be assigned on a first come first serve basis, 1 Issue == 1 PR.
- "Issue Title" and "PR Title should be the same. Include issue number along with it.
- Follow Contributing Guidelines & Code of Conduct before start Contributing.

***********************************************************************
:white_check_mark: **To be Mentioned while taking the issue :**
- Full name : 
- GitHub Profile Link : 
- Participant ID (If not, then put NA) :
- Approach for this Project :
- What is your participant role? (Mention the Open Source Program name. Eg. HRSoC, GSSoC, GSOC etc.)

*************************************************************
Happy Contributing 🚀 

All the best. Enjoy your open source journey ahead. 😎

**Comment by Shrutakeerti** [comment]
Hi, @abhisheks008! I would like to take up this issue.
Full name: Shrutakeerti Datta
Github Profile link : https://github.com/Shrutakeerti
Participant id: N/A
Approach for this project : 
1) Data Preparation: by  Preprocessing the audio data by resampling, normalizing, and extracting relevant features (e.g., MFCCs or spectrograms).
2) Model Selection and Training: Choose an appropriate neural network architecture for audio classification (e.g., CNN, RNN, or hybrid models) and then training and splitting the data
3)Evaluation and Optimization: By evaluating the trained model and optimizing it adjusting the hyperparameters
4) Deployment and Monitoring: Now integrating the trained model  monitoring and checking for the accuracy
What is your participant role: JWOC

**Comment by abhisheks008** [comment]
One issue at a time @Shrutakeerti

**Comment by keshav1441** [comment]
Full name : Keshav Sharma
GitHub Profile Link : https://github.com/keshav1441
Participant ID : NA
Approach for this Project :
To classify audio files, first perform exploratory data analysis (EDA) on the provided dataset to understand its structure and characteristics. Extract relevant features from the audio files, such as Mel-Frequency Cepstral Coefficients (MFCCs). Implement and train multiple classification algorithms, such as Random Forest, Support Vector Machine (SVM), and Convolutional Neural Network (CNN). Compare the performance of these models using accuracy scores and select the best-performing algorithm.
Participant Role : SSOC season 3

**Comment by abhisheks008** [comment]
Implement these models for this dataset,
1. Random Forest
2. Decision Tree
3. Logistic Regression
4. Gradient Boosting
5. XGBoost
6. Lasso
7. Ridge
8. MLP Classifier
9. Support Vector Machine

Assigned @keshav1441
